intellij/src/main/kotlin/motif/intellij/hierarchy/descriptor/ScopeHierarchyDependencyDescriptor.kt

```diff
@@ -32,7 +33,7 @@ open class ScopeHierarchyDependencyDescriptor(
 
     override fun updateText(text: CompositeAppearance) {
         text.ending.addText(method.returnType.simpleName)
-        text.ending.addText(" (" + method.returnType.qualifiedName + ")", getPackageNameAttributes())
+        text.ending.addText(" (" + formatQualifiedName(method.returnType.qualifiedName) + ")", getPackageNameAttributes())
```

Comment:
Kotlin's provides the following [String interpolation syntax](https://kotlinlang.org/docs/reference/idioms.html#string-interpolation):

```kotlin
" (${formatQualifiedName(method.returnType.qualifiedName)})"
``` 

Might make sense to update all existing concatenations in a separate PR.